test/parser/parser_test.cpp

```diff
@@ -221,6 +257,8 @@ TEST_F(ParserTestBase, GarbageTest) {
 TEST_F(ParserTestBase, InsertTest) {
   auto stmts = pgparser.BuildParseTree("INSERT INTO foo VALUES (1, 2, 3), (4, 5, 6);");
   EXPECT_EQ(stmts.size(), 1);
+  stmts = pgparser.BuildParseTree("INSERT INTO foo (id,bar,entry) VALUES (1, 2, 3);");
+  EXPECT_EQ(stmts.size(), 1);
```

Comment:
Is there anything more substantial that we could be testing here?

Reply:
You're right, I could do much better on some of these tests that are missing verification.
I'll put in the checks for the new ones and push an update.